### PR TITLE
added navigation bar and 404 error page

### PR DESCRIPTION
**Pull request by JLODI**
added 404 page not found page
Sign Up link on navigation has a placeholder which leads to the 404 page

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -5,13 +5,21 @@ import {
...
     return (
       <>
-            <Home />
+        <Router>
+          <Navigation />
+          <Switch>
+            <Route exact path="/" component={Home} />
+            <Route component={NotFound}/>
+          </Switch>
+        </Router>
```
Comment (WalkerTexas-coder) [inline app/javascript/components/App.js:22]: Router should be the outermost tag

File: app/javascript/components/components/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                    </NavItem>
+                    <NavItem>
+                    <NavLink className="nav-link" to="/test">
+                        Sign Up
+                    </NavLink>
+                    </NavItem>
+                </Nav>
+                <NavbarText>
+                </NavbarText>
+                </Collapse>
```
Comment (WalkerTexas-coder) [inline app/javascript/components/components/Navigation.js]: Clean up indentation

**Review by WalkerTexas-coder — COMMENTED** [review]
Great work clean up the code and ping me

File: app/javascript/components/App.js (hunk 1 above)
Comment (JLODI) [reply app/javascript/components/App.js:22]: The Router tag is enclosing Navigation and Switch (it's the first tag after the fragment and the last tag before the closing fragment)

Can you please advise what outermost is?

File: app/javascript/components/components/Navigation.js (hunk 2 above)
Comment (JLODI) [reply app/javascript/components/components/Navigation.js]: fixed indentations with the original code for reference

https://reactstrap.github.io/?path=/docs/components-navbar--navbar

**Review by WalkerTexas-coder — APPROVED** [review]
LGTM!

**Pull request merged**